portal-web/test/functional/com/liferay/portalweb/tests/enduser/headless/HeadlessAdminTaxonomyAPI/ExposeErcForCategories.testcase

```diff
@@ -8,11 +8,9 @@ definition {
 	setUp {
 		TestCase.setUpPortalInstance();
 
-		User.firstLoginPG();
```

Comment:
Oh that's interesting, so we don't need to login first in order for JSON/API calls to work? If that's the case...seems like a security vulnerability. Can you double check this test works if run locally without the User.firstLoginPG?

The reason why this test works on CI is because TestCase.setUpPortalInstance signs in. But for local poshi test, `SignIn.signInPortalInstanceSetUp` inside `TestCase.setUpPortalInstance` would usually be skipped.

Reply:
Maybe that is because we pass -u "{username}:{password}" with curl command.

Reply:
I agreed with Ashley, I think in this case we don't need loginPG

Reply:
got it, makes sense